src/python/pants/notes/2.4.x.md

```diff
@@ -4,10 +4,12 @@ See https://www.pantsbuild.org/v2.4/docs/release-notes-2-4 for an overview of th
 
 ## 2.4.1rc0 (Apr 20, 2021)
 
-### Bug fixes
+### User API Changes
 
 * Run MyPy only on specified files, not transitive closure (Cherry-pick of #11936) ([#11938](https://github.com/pantsbuild/pants/pull/11938))
```

Comment:
This was misclassified imo, and it probably should not have been cherry-picked...See Benjy's issue he found with https://github.com/pantsbuild/pants/issues/11957 (which I haven't had time to investigate).

@benjyw do you think we should rever this change in 2.4.x? Imo, yes, it is too breaking to have done after we've already had a stable release.

Reply:
Yes, I tend to think this should not have been cherry-picked. It is not a bugfix by reasonable definitions.

Reply:
Okay, I will revert the change. I agree with you.